specification/redis/resource-manager/Microsoft.Cache/preview/2023-05-01/redis.json

```diff
@@ -2697,6 +3047,167 @@
       ],
       "description": "Parameter required for creating a linked server to redis cache."
     },
+    "RedisCacheAccessPolicyProperties": {
+      "properties": {
+        "provisioningState": {
+          "type": "string",
+          "description": "Provisioning state of access policy",
+          "readOnly": true,
+          "enum": [
+            "Updating",
+            "Succeeded",
+            "Deleting",
+            "Deleted",
+            "Canceled",
+            "Failed"
+          ],
+          "x-ms-enum": {
+            "name": "AccessPolicyProvisioningState",
+            "modelAsString": true
+          }
+        },
+        "type": {
+          "type": "string",
+          "description": "Built-In or Custom access policy",
+          "readOnly": true,
+          "enum": [
+            "Custom",
+            "BuiltIn"
+          ],
+          "x-ms-enum": {
+            "name": "AccessPolicyType",
+            "modelAsString": true
+          }
+        },
+        "permissions": {
+          "type": "string",
+          "description": "Command permissions for the access policy."
+        }
+      },
+      "required": [
+        "permissions"
+      ],
+      "description": "All properties of an access policy.",
+      "type": "object"
+    },
+    "RedisCacheAccessPolicyRequestResponse": {
```

Comment:
>RedisCacheAccessPolicyRequestResponse [](http://example.com/codeflow?start=5&length=37)

I'd just call it RedisCacheAccessPolicy.

Reply:
Kept the name RedisCacheAccessPolicyRequestResponse to maintain naming uniformity with the other definitions of Redis cache access policy assignment (eg. RedisCacheAccessPolicyAssignment contains the base properties of an assignment in its case, different from the request params/response body)

Reply:
I don't see "RedisCacheAccessPolicyAssignment" anywhere in this PR so I wonder what other usages you mean??

Reply:
I still don't think I understand why we don't want to rename RedisCacheAccessPolicyRequestResponse -> RedisCacheAccessPolicy though.

Reply:
or e.g. RedisCacheAccessPolicyDefinition?